src/test/java/edu/unc/lib/boxc/migration/cdm/test/SipServiceHelper.java

```diff
@@ -182,10 +182,12 @@ public void assertObjectPopulatedInSip(Resource objResc, DepositDirectoryManager
      * @param filePaths if withAccessCopies is false, then this is a list of staging paths. Otherwise,
      *      this array should alternate between staging and access paths, staging first. If there is no
      *       access path for an object, then null must be provided.
+     * @return the Bag resource representing the grouped work in the SIP's RDF model
      * @throws Exception
      */
-    public void assertGroupedWorkPopulatedInSip(Resource objResc, DepositDirectoryManager dirManager, Model depModel,
-            String cdmId, boolean withAccessCopies, Path... filePaths) throws Exception {
+    public Bag assertGroupedWorkPopulatedInSip(Resource objResc, DepositDirectoryManager dirManager,
```

Comment:
It is confusing that a method called `assertGroupedWorkPopulatedInSip` returns a `Bag` because it sounds like it should return a Boolean instead. is there any way you can change that to make it clearer?

Reply:
Not sure why I did that, it should be addressed now